tao/tao_bridge/ral/CMakeLists.txt

```diff
@@ -122,6 +122,18 @@ if(${TAO_DCU})
   )
 endif()
 
+if(${TAO_ENABLE_PATINE})
+  list(APPEND RAL_SRCS
+    "tensorflow/compiler/mlir/xla/ral/context/common_context_impl_patine.cc"
+  )
+endif()
+
+if(${TAO_ENABLE_MKLDNN})
+  list(APPEND RAL_SRCS
+    "tensorflow/compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc"
+  )
+endif()
+
 if(${TAO_CPU_ONLY})
```

Comment:
Do we still need the `TAO_CPU_ONLY` flag in the future?

Reply:
I think it's still necessary. For example, Framework like PyTorch would link MKL/oneDNN even for the GPU package.  I think we eventually need this as well especially for the training scenario where we may need cpu compilation & GPU compilation at the same time.